Gruntfile.js

```diff
@@ -15,8 +15,7 @@ module.exports = function (grunt) {
     configFile: __dirname + '/src/server/config/kibana.yml',
 
     nodeVersion: '0.10.35',
-    platforms: ['darwin-x64', 'linux-x64', 'linux-x86', 'windows'],
-    services: [ [ 'launchd', '10.9'], [ 'upstart', '1.5'], [ 'systemd', 'default'], [ 'sysv', 'lsb-3.1' ] ],
+    platforms: ['linux-x64'],
```

Comment:
same as my question below, if we can pass grunt a parameter to only build for linux, this may be unecessary